### PR TITLE
Do not hide an exception, let it throw

### PR DESCRIPTION
**Pull request by sylvain-charton**
and let the consumer catch it and deal with it

### CODE REVIEW

File: src/main/java/com/arangodb/springframework/repository/query/ArangoResultConverter.java [hunk 1]
```diff
@@ -112,18 +112,13 @@ public ArangoResultConverter(final ArangoParameterAccessor accessor, final Arang
...
-		} catch (final Exception e) {
-			e.printStackTrace();
-			return null;
+		if (type.isArray()) {
+			return TYPE_MAP.get("array").invoke(this);
+		}
+		if (!TYPE_MAP.containsKey(type)) {
+			return getNext(result);
 		}
+		return TYPE_MAP.get(type).invoke(this);
```
Comment (abuechler) [inline src/main/java/com/arangodb/springframework/repository/query/ArangoResultConverter.java]: Either throws declaration or something like:

```suggestion
		try {
			if (type.isArray()) {
				return TYPE_MAP.get("array").invoke(this);
			}
			if (!TYPE_MAP.containsKey(type)) {
				return getNext(result);
			}
			return TYPE_MAP.get(type).invoke(this);
		} catch (InvocationTargetException | IllegalAccessException e) {
			throw new ArangoUncategorizedException("Unable to convert the result.", e);
		}
```

**Comment by sylvain-charton** [comment]
Looks like there are CI jobs which can't manage to start their DB instance...

**Review by rashtao — APPROVED** [review]
LGTM, ignore CI failures since they are unrelated to your code

**Comment by rashtao** [comment]
Hi @sylvain-charton ,

Many thanks for contributing. We are obliged by law to have a signed scanned copy of a Contributor License Agreement (CLA) to be able to accept pull requests to the official ArangoDB repository - even for such small changes.

We use an Apache 2 CLA for ArangoDB and its companion projects, which can be found here: https://www.arangodb.com/documents/cla.pdf Just fill in the form, sign and send a scanned copy over to cla@arangodb.com

Thank you

**Comment by rashtao** [comment]
It seems this has been already fixed by https://github.com/arangodb/spring-data/commit/7e948588ac49c48fdcb2767f48f55f0371caccf1

**Pull request closed without merging**